### ISSUE TITLE
Adding credits

### ISSUE THREAD

**Issue by finanalyst** [Raku/doc-website#39]
What should be the policy for adding credits?
Here I am asking exclusively about the website tooling. The credits for adding content to the Documentation should be separate.

There is 
- the `authors` section of META6.json
  - Only allows for a name, no description or profile
- the **about** page
  - Allows for more info about author

Currently we only have initial drafts and this needs to change to include more people
- META6.json only contains my name, but should have other contributors
- **About** was only slightly edited.
 - I also think we should credit the primary contributors to the previous website design, such as Moritz Lenz

Several options:
- Make the `authors` section and about page contain the same names
   - Keeping them synchronised adds hassle.
   - When a new contributor has a PR accepted, they are asked to change the META6 and **about** sections as well.
  - We could create a plugin that takes the META6.json `authors` section and adds it to the **about** page
- Have a distinction of some sort between the META6.json key and about page
  - I'm not keen on this option as I cant see a valid distinction between the two places

**Comment by 2colours** [comment]
I think giving credit to the people who are actively giving something to everybody is an important thing, and I also agree that people like Moritz or JJ (and probably others I don't even know because of newbieness) do deserve to be credited because we might not be here without their work for years.

In the light of this, I definitely don't want to underestimate the importance of credits - however, I feel that the simple nature of the topic could lead to severe bikeshedding so probably it needs to be accepted that everybody will have to make a compromise, if we want to discuss this topic.

By the way, I think this would actually make a good https://github.com/Raku/doc-website/discussions entry.

My opinion is that adding people to META6 `authors` who didn't actually author the distribution, is a bit weird, and so is to make that list bloated. I'm not sure, for one, when a contributor starts to feel like an author. To say that I'm an author because I added some mini Javascript quirk to a template... it is flattering indeed, but frankly it might be a bit of an overstatement. It feels different from Rakudo contributor lists which are like a very objective matter - if you have commits in the source, you are a contributor - but again, nobody would say that all those people are "authors of Rakudo", that would be a bit meaningless.


TL;DR: 1. the topic is valuable but beware of bikeshedding 2. maybe it could move to the Discussions page +1: I would assume that we can go online before ultimately making a decision about this?

**Comment by coke** [comment]
IMO, the authors section in META6,json is for the active module maintainers of doc-website. (For many modules I'd say esp. those doing a release but we haven't determined we're doing releases to support the devops site yet, so...)

The credits page is new; this can include important current (and historic) contributors to both doc-website and doc, but this doesn't need to be everyone who's ever committed.

Finally, doc has CREDITS which is a more exhaustive (but always incomplete) list of people who have code/text that was contributed. The web UI can link to a CREDITS in both repositories on github main. We can/should ask people with PRs to add themselves to CREDITS if they want. Let's create a CREDITS of the same format.

Note: I think it's reasonable to replace the credits page entirely with a link to the CREDITS files.

**Comment by coke** [comment]
Added CREDITS in #44, with everyone who has had a PR of some kind, plus the two callouts made above for JJ & Moritz. Please add or expand as needed.
